addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -132,6 +148,15 @@ const Wysiwyg = Widget.extend({
             plugins: options.editorPlugins,
         }, editorCollaborationOptions));
 
+        document.addEventListener("mousemove", this._signalOnline, true);
+        document.addEventListener("keydown", this._signalOnline, true);
+        document.addEventListener("keyup", this._signalOnline, true);
```

Comment:
Why capture ?

Reply:
As this event is important, using capture improve the likelihood of the handler of being called (reducing the chance that a handler would stop the propagation).